app/services/hts_service/dashboard.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+# HTS Service
+module HTSService
+  # Dashbiard Class
+  class Dashboard
+
+  def self.daily_statistics(start_date,end_date)
+
+art = Observation.joins("INNER JOIN concept_name ON concept_name.concept_id = obs.concept_id")
```

Comment:
ART outcome is saved as a value coded observation under the concept of "ART Referral" with possible value coded values of
              'Link',
                'Refused',
                'Died',
                'On ART',
                'Unknown'